jinahub/indexers/searcher/compound/FaissPostgresIndexer/README.md

```diff
@@ -34,9 +35,54 @@ with:
 
 The folder needs to contain the data exported from your Indexer. Again, see [docs](https://docs.jina.ai/advanced/experimental/indexers/).
 
+Check [integration tests](https://github.com/jina-ai/executors/tree/main/tests/integration/psql_dump_reload) for an example on how to use it.
+
 #### Direct import from PSQL
 
-Alternatively, you can use the import from PSQL feature. Check the [integration test](https://github.com/jina-ai/executors/tree/main/tests/integration/psql_import).
+Alternatively, you can use the import from PSQL feature. 
+
+This allows you to bypass the expensive writing to disk operation. It also removes the complexities that can arise out of doing this in a cloud environment, where you would need to mount the volume in one instance, and then migrate it, and then perform the import. 
+
+Direct import from PSQL can be done in two ways:
+
+1. Via *snapshot*. This guarantees consistency between the shards, but at the cost of speed and extra disk usage. This creates a duplicate of your data in another table, at a specific moment. This way all the shards import from the same view/version of your data.
```

Comment:
Promote direct delta. But with a warning of eventual consistency and recommend snapshot if that's a problem for the user.